### ISSUE TITLE
The terminal provider jansi has been deprecated, check your configuration

### ISSUE THREAD

**Issue by liangcha385** [Dreeam-qwq/Leaf#51]
```
5月 03, 2024 7:29:59 上午 org.jline.utils.Log logr
警告: The terminal provider jansi has been deprecated, check your configuration. This warning can be disabled by setting the system property org.jline.terminal.disableDeprecatedProviderWarning to true.
```
[Jansi](http://fusesource.github.io/jansi/) is a small java library that allows you to use [ANSI escape codes](http://en.wikipedia.org/wiki/ANSI_escape_code) to format your console output which works even on Windows. 
But it is in an abandoned state and this situation has occurred on both my laptop(Windows 10 22H2) and server(Windows Server 2019 Datacenter 1809).

**Comment by HaHaWTH** [comment]
You can disable this warning by adding -Dorg.jline.terminal.disableDeprecatedProviderWarning=true in your jvm args to disable the warning temporarily.
Will assign to Dreeam :/

**Comment by Dreeam-qwq** [comment]
Fixed in 555e5a4425fffc28d5c392e9cad28da03f312fed

**Comment by liangcha385** [comment]
good job : )
